node/antieframework.js

```diff
@@ -113,38 +112,22 @@ var AntieFramework = function(configPath, frameworkPath) {
      * @param string pageStrategy The page strategy used by this device.
      * @param string element The page strategy property to return (Sub-directory of {{_configPath}}/config/pagestrategy/{{pageStrategy}}/{{element}}
      * directory).
-     * @param string default The default value to return if the page strategy does not contain the requested element.
      * @return string An element (property) of the page strategy or the default value.
      */
     var getPageStrategyElement = function(pageStrategy, element) {
```

Comment:
I suppose it depends on whether we tell people to use the tal launcher. If we aren't telling people to do that then I suppose it doesn't matter